### ISSUE TITLE
[SUBMIT]

### ISSUE THREAD

**Issue by bikramkawan** [strapi/community-content#989]
**My resource**

This article is for creating multitenancy strapi app which I have been using for 10+ clients. Reader will be able to run multiple strapi app using single code base.
Here is article I have published.
https://medium.com/@bikramkawan/how-to-run-strapi-as-multitenancy-multi-tenant-app-to-host-multiple-website-with-single-codebase-4dde5f9de6d
**My content is**

- [x] A Strapi Tutorial / Guide / How to article
- [ ] About Headless, CMS, javascript, open source, or jamstack
- [ ] Other, specify

**What do you all think?**
❤️ I love it!
🚀 I can help you!

Thank you for submitting your article proposal.

**Comment by dessireugarte** [comment]
Hello @bikramkawan 👋 
Since you already wrote the article on another platform but in v3, would this one be in v4? 
@PaulBratslavsky @Mcastres what do you think?

**Comment by Mcastres** [comment]
I'm not an expert on Multi-tenancy but it seems too easy to tackle this popular feature 🤔 Maybe you can provide more information on this @bikramkawan :)

**Comment by bikramkawan** [comment]
@Mcastres I can start writing for the v4 if that helps. It will be same principle I have written for V3.  Let me know.

**Comment by Mcastres** [comment]
Yes but I would consider this as an approach of multi-tenancy more than an `official` multi-tenancy tutorial

**Comment by bikramkawan** [comment]
@Mcastres yes I agree. This is the work around not the official way or absolute way.

**Comment by Mcastres** [comment]
As long as the title mention it it sounds good to me then!

**Comment by dessireugarte** [comment]
Would you like me to assign the topic to you @bikramkawan?

**Comment by dessireugarte** [comment]
Great! 
If this is your first article, can you please register your author details [here](https://strapi.io/write-for-the-community) by filling out the form? Then please scroll to the FAQ section and review the article [guidelines](https://github.com/strapi/community-content/blob/master/tutorials/GUIDELINES.md), how to submit an article, issue an invoice, and get paid.
You can start writing your draft. We use Dropbox Paper to submit and review drafts. Please duplicate and use this [article template](https://www.dropbox.com/scl/fi/7o4rzejzmla65hygplkai/My-First-Strapi-Article.paper?dl=0&rlkey=zoc4r7k3iw7titgsxp8vrt55x). 
Once your article is ready, please share the link here so we can review it.

**Comment by bikramkawan** [comment]
Hi @dessireugarte , Could you please check here.
https://www.dropbox.com/scl/fi/3mk04c3bkgq3atkonsbcz/_-Strapi-v4.paper?dl=0&rlkey=yqxor8cssi7zjmnogmgud7bjf

**Comment by dessireugarte** [comment]
Hello @bikramkawan! Thanks for submitting the draft. @PaulBratslavsky will review it and we will give you feedback to publish it hopefully by the end of next week.

**Comment by dessireugarte** [comment]
Also, @bikramkawan could you please fill out the form on [this](https://strapi.io/write-for-the-community) page? I left some comments in your draft.

**Comment by PaulBratslavsky** [comment]
@bikramkawan I left some comments in the drop box file.

**Comment by bikramkawan** [comment]
@PaulBratslavsky and @dessireugarte Thanks for the comment. I have updated based on your suggestion.

**Comment by dessireugarte** [comment]
Hello @bikramkawan  👋 
Your article is [live](https://strapi.io/blog/how-to-build-a-pseudo-multi-tenant-application-in-strapi)! 🎉
Please register to our forum by clicking on the discussion at the bottom of your post, so you automatically receive notifications when someone adds comments. Also please follow the [guidelines](https://strapi.io/write-for-the-community) to issue and submit an invoice. 

You should register yourself **once** as an "Individual Vendor" in this [form](https://docs.google.com/forms/d/e/1FAIpQLScIl6QgX72c5hiVvdwjkIOXju5hkIHU5-Sz3FU8mYQRFsqJFQ/viewform), please let me know if you have any questions about the form. Just be aware that in the W-8BEN form:
- 5 is not mandatory
-  6a & 6b are only necessary if you have a tax number in your own country. 

Then you can submit your invoice [here](https://docs.google.com/forms/d/e/1FAIpQLSeD59R4fR5dEJIRh4JhUpBlKAi9yX5YeJIMsn-3gUFVj-bMjg/viewform) for $200, and soon you will receive your payment.
If you want to promote your article, [here](https://strapi.notion.site/Promotion-Template-for-Writers-1645b77a97a640d2a32047297b6bc3f0) are some ideas and templates that might help you!
Please feel free to reach out to me if you have any questions!
